libraries/chain/services/proposal_executor.cpp

```diff
@@ -23,6 +23,14 @@ dbs_proposal_executor::dbs_proposal_executor(database& s)
     evaluators.register_evaluator<registration_committee::proposal_change_quorum_evaluator>();
     evaluators.register_evaluator<registration_committee::proposal_exclude_member_evaluator>(
         new registration_committee::proposal_exclude_member_evaluator(services, removed_members));
+
```

Comment:
Proposal executor is not a service. It should be removed from services folder and created via separate fabric.